### DISCUSSION TITLE
Hide featured image in the post

### DISCUSSION THREAD

**Discussion by annaisdevk** [GetPublii/Publii#1404] [Help] [answered]
Hi,

In the posts I have added an image in the 'Featured Image' option. Is it possible that this image is only shown in the post preview, but not in the post?

**Comment by example-user30** [comment]
Do I understand you want to hide the featured image when the post is viewed on the single page but want to see it when the list of posts are displayed?  What theme are you using?

**Reply by annaisdevk** [reply]
Yes. I want the featured image to be hidden when the post is viewed on the single page, but displayed when the list of posts is shown.

The theme used is [Taste](https://marketplace.getpublii.com/themes/taste/).

**Reply by example-user30** [reply, marked as answer]
Go to Tools & Plugins -> Custom CSS

Add the following CSS:

.post__featured-image {
    display: none;
}

![shot_240228_155615](https://github.com/GetPublii/Publii/assets/156127535/2ffd575f-0ab6-4b07-8f39-4d1829f8c94b)

Save & Preview.

**Reply by candidexmedia** [reply]
Just wanted to chime in: if you wanted a post settings option to hide the cover photos of some posts but not others, see: https://forum.getpublii.com/topic/how-to-remove-featured-picture-from-posts/#post-8596

**Reply by example-user30** [reply]
That is really slick!

**Reply by rgblack316** [reply]
Too bad it doesn't work on every theme. :-( I'm trying to hide featured images on posts in the Editorial-2 theme and having no luck at all.

**Reply by gorgenarrows** [reply]
The site I'm working on uses a modified Persona theme. I added a custom post configuration that I call "Featured image location" that can take the values:
- above the title
- below the title
- on the left
- on the right
- hidden
I ended up writing a couple of "Partials" that I include in appropriate places. It wasn't too difficult to set up, and I find it quite handy.

For example, in my "post.hbs" template, the code for the "above" part looks like this:
```
   {{#post}}
      <article class="content">      
         {{#contains @config.post.featuredImagePosition 'above'}}
            {{>featured-image-block context=@config.post}}
         {{/contains}}
         <header class="content__inner content__header">
            <h1 class="content__title">
```

**Reply by candidexmedia** [reply]
@rgblack316 It works, you just need to use the right CSS class for your theme. You can use your [browser's Inspect Tool](https://blog.hubspot.com/website/how-to-inspect) to find it:

![image](https://github.com/user-attachments/assets/c45e5489-1c93-49fa-adf3-5d98bcfaca86)

Once found, go to your Custom CSS (in the Publii GUI) or open main.css (using a text editor), and add/modify the following:

```css
.image.main {
  display: none;
}
```

![image](https://github.com/user-attachments/assets/a6c4237d-7b68-4fc0-8314-f5bb1cb669da)
